types/electra/attestation.yaml

```diff
@@ -0,0 +1,18 @@
+Electra:
+  Attestation:
+    type: object
+    description: "The [`Attestation`](https://github.com/ethereum/consensus-specs/blob/v1.5.0-alpha.3/specs/electra/beacon-chain.md#attestation) object from the CL spec."
+    required: [aggregation_bits, data, signature, committee_bits]
+    properties:
+      aggregation_bits:
+        $ref: "../primitive.yaml#/BitList"
+        description: "Attester aggregation bits."
+      data:
+        $ref: "../attestation.yaml#/AttestationData"
```

Comment:
Since you added a description to every other field, can you add one to `data` too?

Reply:
The default description of `AttestationData` should be sufficient as it has not been modified in electra

https://github.com/ethereum/beacon-APIs/blob/672e03e25ace85a3bacaea553fbf374f4f844435/types/attestation.yaml#L46-L48

Reply:
I'm happy with that default description...